data/locale/attributes.adoc

```diff
@@ -245,3 +245,24 @@ ifeval::["{lang}" == "zh-CN"]
 :version-label: 版本
 :warning-caption: 警告
 endif::[]
+
```

Comment:
I think you need to remove this blank line (see instructions at the top of the line)